web/src/compose_validate.ts

```diff
@@ -370,7 +370,7 @@ export function warn_if_mentioning_unsubscribed_group(
     let any_member_subscribed = false;
     for (const user_id of group_members) {
         if (
-            stream_data.is_user_subscribed(stream_id, user_id) &&
+            (await stream_data.maybe_fetch_is_user_subscribed(stream_id, user_id, false)) &&
```

Comment:
Because the stream is the same for all of these, there's no potential benefit to doing the `await`s in parallel, since the data set to fetch is the same for every user.